src/GSF.CarbonAware/src/Json/DateTimeOffsetConverter .cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace GSF.CarbonAware.Json
```

Comment:
Hi @in4margaret. Is this is an issue only with Json output format? Or is it related with the Json static data? If it is the latter, would it be this belonging to the Json Data Source instead?

Reply:
It is not only for JSON static data file.  Currently in integration tests with ElectricityMaps in Mocker dates with undefined timezone returns data in the timezone that is local to mocked server. It is a good example of a data source that provide dates in timezone specific format. Our cli and WebAPI should return the same format regardless of underlying datasource format. That is why we need to specify output timezones ourselves.

Reply:
> It is not only for JSON static data file. Currently in integration tests with ElectricityMaps in Mocker dates with undefined timezone returns data in the timezone that is local to mocked server. It is a good example of a data source that provide dates in timezone specific format. Our cli and WebAPI should return the same format regardless of underlying datasource format. That is why we need to specify output timezones ourselves.

Got it, thanks for the explanation. One more question: Is this a Mock/Integration test issue, only? Or do you see the same behavior using the datasources directly?